spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/index.adoc

```diff
@@ -27,30 +27,20 @@ endif::backend-html5[]
 
 // ======================================================================================
 
-[[overview]]
-= Spring Cloud Data Flow for Cloud Foundry
-
-This project provides support for orchestrating the deployment of Spring Cloud Stream applications to Cloud Foundry.
-
-include::overview.adoc[]
-
-include::{dataflow-asciidoc}/architecture.adoc[]
-
 include::getting-started.adoc[]
 
+include::{dataflow-asciidoc}/applications.adoc[]
+include::{dataflow-asciidoc}/architecture.adoc[]
 include::{dataflow-asciidoc}/configuration.adoc[]
 
+include::{dataflow-asciidoc}/shell.adoc[]
 include::{dataflow-asciidoc}/streams.adoc[]
-
 ifndef::omit-tasks-docs[]
 include::{dataflow-asciidoc}/tasks.adoc[]
-endif::omit-tasks-docs[]
-
 include::cf-tasks.adoc[]
-
+endif::omit-tasks-docs[]
```

Comment:
why did you move this? seems like the original intent was to only exclude the task docs from core if `omit-tasks-docs` was set